front/components/assistant/conversation/InputBar.tsx

```diff
@@ -609,6 +614,30 @@ export function AssistantInputBar({
                         }}
                       />
                     ))}
+                    <div className="flex flex-row justify-between border-t border-structure-100 px-3 py-2">
```

Comment:
Shouldn't this be a justify-center with a gap-x ?

Reply:
From the figma below

Reply:
![image](https://github.com/dust-tt/dust/assets/5437393/ff8fc6e3-eb54-403d-b3c7-ef15750ad3b0)

Reply:
for me this is justify between and since ed approved I think we're good :)